### ISSUE TITLE
Website Accessibility-WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (.mxnavbar__brand)

### ISSUE THREAD

**Issue by A11Ybot-OP** [matrix-org/matrix.org#1208]
#### Issue
`Ensures links have discernible text` ([link-name](https://accessibilityinsights.io/info-examples/web/link-name))
#### Target application
[Matrix.org](https://matrix.org/)
#### Element path
.mxnavbar__brand
#### Snippet
`<a href="/" class="mxnavbar__brand w-nav-brand w--current"><img src="/images/matrix-logo.svg" alt="" class="mxnavbar__logo"></a>`
#### How to fix
    Fix all of the following:
      Element is in tab order and does not have accessible text
    
    Fix any of the following:
      Element does not have text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
#### Environment
Microsoft Edge version 96.0.1054.62
